core/shared/src/main/scala/cats/parse/extra/DefaultParser0.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2021 Typelevel
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package cats.parse.extra
+
+import cats.parse.{Parser, Parser0}
+
+/** Typeclass for "has a Parser0"
+  *
+  * This is primarily provided to help keep track of `Parser0` instances, and as such the omission
+  * of `cats.Functor` instances is intentional.
+  * @tparam A
+  */
+trait DefaultParser0[+A] {
+  def parser0: Parser0[A]
```

Comment:
Can we add plumbing through parse and parseAll here so you can do: `DefaultParser[Foo].parseAll(s)`

Reply:
Easy enough 👍🏻

Reply:
I don't see this change yet. Did you reconsider this?

Reply:
Looks like it's up, my guess is it took a couple minutes to propagate to the UI

Reply:
this was a good idea 🙌